deep_speech_2/README.md

```diff
@@ -331,7 +335,30 @@ python tune.py --help
 ```
 or refer to `example/librispeech/run_tune.sh`.
 
+## Running in Docker Container
+
+Docker is an open tool to build, ship, and run distributed applications in an isolated environment. A Docker image for this project has been provided in [hub.docker.com](https://hub.docker.com) with all the dependencies installed, including the pre-built PaddlePaddle, CTC decoders, and other necessary Python and third-party packages. This Docker image requires the support of NVIDIA GPU, so please make sure its availiability and the [nvidia-docker](https://github.com/NVIDIA/nvidia-docker) has been installed.
```

Comment:
open tool ? Did you mean open source tool ?

Reply:
Yes, modified.